### ISSUE TITLE
MAST differential expression not working

### ISSUE THREAD

**Issue by gufranca** [satijalab/seurat#589]
Dear Seurat developers,

I am having an issue with detecting differentially expressed genes using "MAST" method. It returns no differentially expressed genes, no matter which dataset I am using. Interestingly, this started to happen after updating Seurat (2.3.3) and R (3.5.0). Previously it was working well. All my previous analyses were done using MAST and now I can't reproduce anymore. Please, could anyone help to fix this issue? 
Thank you,
Gustavo
```R
>FindAllMarkers(object = data, only.pos = TRUE, test.use = "MAST", min.cells.gene = 5)
Warning message:
In FindAllMarkers(object = data, only.pos = TRUE, test.use = "MAST",  :
  No DE genes identified.

> sessionInfo()
R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.10

Matrix products: default
BLAS: /usr/local/bin/R/3.5.0/lib/R/lib/libRblas.so
LAPACK: /usr/local/bin/R/3.5.0/lib/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] AnnotationDbi_1.42.1 IRanges_2.14.10      S4Vectors_0.18.3     Biobase_2.40.0       BiocGenerics_0.26.0  Seurat_2.3.3        
[7] Matrix_1.2-14        cowplot_0.9.2        ggplot2_2.2.1`
```

**Comment by huyqdinh** [comment]
It was not only MAST, all other methods/params returned like that, weirdly it was also with the old Seurat version when I tried to reverse to see if version matters.

**Comment by gufranca** [comment]
It seems related to R 3.5 version. When I upgraded R to 3.5 it stopped working, with 3.4 it still works fine for me. Hopefully this issue can be fixed to work on R 3.5.

**Comment by satijalab** [comment]
A couple users have reported something similar - we're not sure if its an issue with Seurat imports or MAST imports, but either way #662 reports a simple solution.
